examples/js/loaders/OBJLoader.js

```diff
@@ -2,6 +2,11 @@
  * @author mrdoob / http://mrdoob.com/
  */
 
+THREE.ParseException = function(message) {
```

Comment:
Why not doing this instead? `throw new Error( "Unexpected line: " + line );` Do we need a new class in the `THREE` namespace just for this?